src/main/java/com/smalaca/messagesender/repository/inmemory/InMemoryMessageRepository.java

```diff
@@ -32,4 +32,14 @@ public void add(Message message) {
     public boolean exists(Message message) {
         return messages.contains(message);
     }
+
+    public void delete(String messageId) {
```

Comment:
dobrze byłoby dodać annotację @Override. 
Tutaj więcej na ten temat [why use?](http://javarevisited.blogspot.com/2012/11/why-use-override-annotation-in-java.html)